src/pip/index.js

```diff
@@ -43,6 +43,17 @@ module.exports.create = function createPIPService(datapath, layers, localizedAdm
   // ie - _.intersection([1, 2, 3], [3, 1]) === [1, 3]
   layers = _.intersection(defaultLayers, _.isEmpty(layers) ? defaultLayers : layers);
 
+  // further refine the layers by filtering out layers for which there is not metafile
+  layers = layers.filter(layer => {
+    const filename = path.join(datapath, 'meta', `wof-${layer}-latest.csv`);
+
+    if (!fs.existsSync(filename)) {
+      logger.warn(`unable to locate ${filename}`);
```

Comment:
If we make this a warning, all our builds will silently proceed with missing admin lookup data. At least now the build would hang since the admin lookup data will never finish loading. Kinda like the case of being unable to load individual files with the whosonfirst importer, we might want to either make this a fatal error or at least have the option to make it one.

Reply:
Switched to an error message.